### DISCUSSION TITLE
Password reset with Supabase & Flutterflow web app

### DISCUSSION THREAD

**Discussion by Empirome** [supabase/supabase#16014] [Questions]
I'm experiencing an issue with the supabase integration in FlutterFlow in my mobile/web app.

This is the work flow:
1. In the mobile app the user can push a "Forgot password" button.
2. This will trigger the Supabase Auth "Password Reset" action in the FlutterFlow mobile app.
3. The user receives the Email from Supabase. This is the URL configuration I have set in Supabase:
   - Site URL: https://appname.flutterflow.app/
   - Redirect URLs:
      - https://appname.flutterflow.app/*
      - https://appname.flutterflow.app/resetpassword

4. Clicking the link loads the correct page but there is an auth error which prevents the user from saving the new password. The debug message says that the user is not logged in. This is what I briefly see in the browser after clicking on the link in the Email:
`appname.flutterflow.app/resetpassword#error=unauthorized_client&error_code=401&error_description=Email+link+is+invalid+or+has+expired`

Not sure what settings I need to use in supabase to make this work. Does it even work with web apps hosted with FlutterFlow?

**Comment by TalMikey** [comment]
Did you find a solution?

**Comment by tierraskyllc** [comment]
Did you find a solution?
